### ISSUE TITLE
WCAG 2.0 Compliance for Logo Components

### ISSUE THREAD

**Issue by timwright12** [10up/wp-component-library#72]
Per our slack conversation in **#10up-frontend-team** about WCAG standards in image ALT text being the only content within a link.

**Primary Error:**
Official word from WCAG 2.0 is that: “image alt text cannot be the primary text of a link"

**Secondary Error (when using a heading):**
"Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags."

Reference: https://www.w3.org/TR/WCAG20/#perceivable
